FRINXio/frinx-frontend
# move inventory queries to new proxy server

Moved all inventory server queries to the new graphql proxy server queries.

HOW TO TEST:
- make sure you have the new proxy server running (localhost:5555)
- have a running inventory server on `story-new-graphql-proxy` branch
- click around the inventory UI (just running type-check should be enough though)
